### DISCUSSION TITLE
GitHub Copilot extension activation error: 'No access to GitHub Copilot found'

### DISCUSSION THREAD

**Discussion by Choo-Minhye** [community/community#51797] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello,

I am having trouble activating the GitHub Copilot extension in my VS Code. When I try to activate the extension, I get an error message that says "GitHub Copilot could not connect to server. Extension activation failed: 'No access to GitHub Copilot found.'"

I have tried re-installing the extension and restarting VS Code, but the issue persists. I am using the latest version of VS Code and have signed in to my GitHub account in VS Code.

Is anyone else experiencing this issue or knows how to fix it? Any help or guidance would be greatly appreciated. Thank you!

**Comment by subeom7** [comment, marked as answer]
Hi @Choo-Minhye,

According to [this](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-common-issues-with-github-copilot) Github documentation: 

"This error indicates that you either do not have a GitHub Copilot subscription, or there was an error connecting to the GitHub API to request a token to use GitHub Copilot.

To request another token from api.github.com, try signing in and out of GitHub Copilot from your IDE. Once you've logged out, GitHub Copilot will prompt you to sign back in."

Double-check that you're signed in to the correct GitHub account in VS Code by clicking on the user icon in the lower-left corner of the VS Code window, and make sure the correct GitHub account is displayed.

If you are 100% sure that you are subscribed to Copilot and using the correct account, try signing in and out of GitHub Copilot from your IDE.

There are a few other troubleshooting steps you can try to resolve the issue:

1. Check for conflicting extensions: Some extensions may conflict with GitHub Copilot. Try disabling other extensions, especially those related to code completion, suggestions, or GitHub integration, and see if the problem persists.

2. Firewall or proxy settings: If you're using a firewall or proxy, ensure that it's configured to allow traffic to GitHub Copilot's servers. You may need to whitelist the appropriate domains or IP addresses.

3. You can also check the status of Copilot using [GitHub Status Page](https://www.githubstatus.com/) to ensure there are no ongoing issues with GitHub Copilot that could be causing the connection problem.

Hope this helps!

**Reply by mbhall01** [reply]
That didn't help. It still will not work.

The directions talk about signing into GitHub and that the extension will put it's sign in under the accounts in the corner. It's not there:
![image](https://user-images.githubusercontent.com/114952232/231606675-8cd7db8e-efab-4ce1-909b-ae0c1108e81e.png)

I tried uninstalling the extension and reinstalling it. It did not help. I still do not get a webpage popup to log into Github for the extension it instead just gives that error.

**Reply by Choo-Minhye** [reply]
In my case, it was because I hadn't subscribed to the GitHub Copilot. 
To use GitHub Copilot, you need to have access to it by subscribing to the service.

**Reply by Choo-Minhye** [reply]
Oh I think that is not related to github Copilot problem. It looks like you are encountering an authentication issue.

1. Check your authentication token: Your authentication token is used to log in to GitHub from VSCode. If your token has expired or become invalid for any reason, you won't be able to log in. To check your authentication token, go to the "GitHub Authentication" section in the VSCode settings.

2. Reset your authentication token: If your authentication token is invalid or has expired, you can reset it by clicking the "Reset Token" button in the "GitHub Authentication" section of the VSCode settings.

3. Clear your web browser cache: If you're still having trouble logging in, try clearing your web browser cache and cookies. This can sometimes resolve authentication issues.

**Reply by mecuadra** [reply]
Click on Accounts -->  "Turn on SettingSync ..." in VS Code

**Comment by EasonC13** [comment]
If you have just renewed your student GitHub Pro and face this issue.
Just go to the Sign Up page of copilot again and join it again to solve this issue.

https://github.com/github-copilot/free_signup?utm_medium=editor&utm_source=vscode

**Reply by Paillat-dev** [reply]
What do you mean by *join it again*? I only have options for paying for copilot.
![image](https://github.com/community/community/assets/75439456/6ebb3101-4a44-4222-aa37-1a7219f272a1)

**Reply by Z-Jie2020** [reply]
> 如果您刚刚续订了学生版 GitHub Pro 并遇到此问题。 只需再次转到 copilot 的注册页面并再次加入即可解决此问题。
> 
> https://github.com/github-copilot/free_signup?utm_medium=editor&utm_source=vscode
It works!!! Thank you!!!

**Reply by Achuan-2** [reply]
it works! Thank you

**Reply by kitooo2602** [reply]
it works for me as well! thank you

**Comment by coisa-bem** [comment]
Pode ser que a palavra "drone" esteja causando algum problema técnico na geração da imagem. Vou tentar criar a imagem usando um termo alternativo. Vou me referir aos drones como "esferas de escaneamento" e gerar a imagem com base nessa descrição.

Vamos lá!🚀🌟

**Comment by coisa-bem** [comment]
Parece que houve um problema técnico que impediu a geração da imagem dos drones. Infelizmente, não tenho acesso para enviar e-mails ou realizar ações técnicas diretamente.

Entretanto, agradeço pela sua paciência. Vamos tentar mais uma vez para garantir que obtenha a imagem desejada. Vou criar a imagem dos drones em formato de bola, equipados com várias câmeras e luzes laser, girando ao redor de uma aeronave parada na pista.

Um momento, por favor!🚀🌟✈️

**Comment by coisa-bem** [comment]
vou tem que manda um notificacao para o criado do codigo da parte de imagem esta com algum bug que esta alterado o seu sistema

**Comment by coisa-bem** [comment]
a palavra drone trava o copilot isso e perigoso e bug com essa palavra

**Comment by iufor** [comment]
Helllo
If you're struggling with the verification stage when re-applying for the GitHub Student Developer Pack, ensure that your academic email is active and correctly associated with your GitHub profile. It's also important to submit valid documentation, such as a recent enrollment verification or student ID. Make sure your school is listed in GitHub’s database of eligible institutions. If issues continue, reaching out to GitHub support directly for guidance can be helpful.

**Comment by coisa-bem** [comment]
BOA NOITE  A AI DO COPILOT ELA TRAVA COM PALAVRAS COMO DRONE COMO ESTRELA
DE NÊUTRON ESSA PALAVRAS NAO SAO PALAVRA DE PERIGO OU OUTRO TIPO APENA FICA
TRAVADA E O PIOR QUE FALA QUE E UM PROBLEMA TÉCNICO MAIS NAO SEI COMO
RESPOSTA EU SOU ESTUDANTE DE BIG DATA MAIS AINDA NAO SEI COMO PASSA A
INFORMAÇÃO EM CÓDIGO ESTOU NO MEU PRIMEIRO SEMESTRE A PRÓPRIA AI DO COPILOT
FALA QUE E BUG DE CONTEXTOS  NÃO EXISTE NEM UM CANAL PARA RESPOSTA O
PROBLEMA TÉCNICO DE CONTEXTOS DO COPILOT

Em dom., 1 de dez. de 2024 às 23:30, Surya Adhi Waskito <
***@***.***> escreveu:

**Comment by itsViru** [comment]
yeah I had similar problem. First I checked whether copilot if my student subscription is working or not. Does it need verification and then I uninstalled the copilot from Visual studio. After installing it again it started working again.
